runtime/caml/mlvalues.h

```diff
@@ -205,7 +205,7 @@ bits  63        (64-P) (63-P)        10 9     8 7   0
 #define Op_val(x) ((value *) (x))
 #define Op_atomic_val(x) ((atomic_value *) (x))
 /* Fields are numbered from 0. */
-#define Field(x, i) (((value *)(x)) [i])           /* Also an l-value. */
+#define Field(x, i) (((volatile value *)(x)) [i]) /* Also an l-value. */
```

Comment:
`Field(x, 0)` and `Op_val(x)` previously had the exact same behavior, but `Field` is now a volatile cast whereas `Op_val` went unchanged in this PR.

Was this on purpose?

Reply:
I had missed it, but perhaps https://github.com/ocaml/ocaml/issues/11426 will take care of that.